pl_bolts/models/autoencoders/basic_ae/basic_ae_module.py

```diff
@@ -13,7 +13,7 @@ class AE(LightningModule):
 
     def __init__(
             self,
-            datamodule: LightningDataModule = None,
```

Comment:
datamodule shouldn't be input, its metadata should

Reply:
then move it out completely